index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="styles.css">
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.0/css/fontawesome.min.css" integrity="sha384-z4tVnCr80ZcL0iufVdGQSUzNvJsKjEtqYZjiQrrYKlpGow+btDHDfQWkFjoaz/Zr" crossorigin="anonymous">
+</head>
+<body>
+    <section class="hero-section">
+        <div class="headline-img"></div>
+        <div class="secondary-cta">
+            <ul class="font-face second-cta container">
+                <li class="s-list"><a class="social" href="#"><img src="./images/twitter.svg" alt="twitter"></a></li>
+                <li class="s-list"><a class="social" href="#"><img src="./images/facebook.svg" alt="facebook"></a></li>
+                <li class="s-list"><a href="#">English</a></li>
+                <li class="s-list"><a href="#">My Page</a></li>
+                <li class="s-list"><a href="#">Logout</a></li>
+            </ul>
+        </div>
```

Comment:
- [ ] On mobile screen size, this part of the navbar should become hidden. But in your own case, the part still appears on mobile size. Kindly look into that and fix.

### Your Design

![your-navbar-issue](https://user-images.githubusercontent.com/26597432/197183839-cf73b254-0843-4069-b769-0306c183059f.png)


### Guiding Template

![temp-navbar](https://user-images.githubusercontent.com/26597432/197183912-ad2deeee-361e-44dc-81d3-cc6fec211f6e.png)